app/assets/stylesheets/application.scss

```diff
@@ -130,3 +130,26 @@
   color: black;
 }
 
+.zero-margin {
```

Comment:
Can we use camelcase for the class names. I think that it was the style we were using across all the codebase. Same for all below.